### DISCUSSION TITLE
How do I increase the num_workers value correctly?

### DISCUSSION THREAD

**Discussion by KatieBelli** [voicepaw/so-vits-svc-fork#698] [Q&A]
I always get this warning before the training starts:

> C:\Users\...\AppData\Local\Programs\Python\Python39\lib\site-packages\lightning\pytorch\trainer\connectors\data_connector.py:430: PossibleUserWarning: The dataloader, val_dataloader,    warnings.py:109
>                     does not have many workers which may be a bottleneck. Consider increasing the value of the `num_workers` argument` (try 8 which is the number of cpus on this machine) in the `DataLoader` init to
>                     improve performance.
>                       rank_zero_warn(

I don't understand this error because I changed the value of num_workers to 8 in the config file before training.

**Comment by TheMeIonGod** [comment]
I would like to know the same thing.

**Comment by single-serval** [comment]
same, and there is only a warning code in the directory/file it asks for. Mine is telling me to up to 15 but I can't find a way to actually do that and have it read it

**Comment by kevinlinxc** [comment]
Yep, same question, why is this so weird? It should be an easy to answer question

**Comment by 34j** [comment]
You can edit the code to increase num_workers (which has no meaning, please refer to the PyTorch docs) or send a PR to suppress this annoying warning https://github.com/Lightning-AI/pytorch-lightning/pull/10215/files
